app/services/sign_in/attribute_validator.rb

```diff
@@ -161,11 +162,9 @@ def check_lock_flag(attribute, attribute_description, code)
       handle_error("#{attribute_description} Detected", code, error: Errors::MPILockedAccountError) if attribute
     end
 
-    def check_id_mismatch(id_array, id_description, code)
+    def check_id_mismatch(id_array, id_description, code, error: Errors::MPIMalformedAccountError)
```

Comment:
Can you use the `prevent_auth: true` similar to how `attribute_mismatch_check` does it? I only suggest this so it matches how that set of checks deal with non-auth-blocking checks